packages/flutter/test/rendering/rendering_tester.dart

```diff
@@ -33,24 +33,22 @@ class TestRenderingFlutterBinding extends BindingBase with SchedulerBinding, Ser
     renderView.compositeFrame();
     if (phase == EnginePhase.composite)
       return;
-    if (SemanticsNode.hasListeners) {
-      pipelineOwner.flushSemantics();
-      if (phase == EnginePhase.flushSemantics)
-        return;
-      SemanticsNode.sendSemanticsTree();
-    }
+    pipelineOwner.flushSemantics();
+    if (phase == EnginePhase.flushSemantics)
+      return;
```

Comment:
this changed the semantics. Now you're always sending the semantics tree. That's probably fine, but may affect some of the tests.

 (Also, there's no point having that last if test, better make it an assert.)